src/js/index.js

```diff
@@ -76,6 +82,14 @@ const lightColors = [
   '#E0E4ED',
   '#edfff2',
   '#EDF6FF',
+  '#BED1FA',
+  '#F2F6FF',
+  '#F9F9F9',
+  '#BDD1FA',
+  '#898FA2',
+  '#DCDFE7',
+  '#F8FAFE',
+  '#D5D7DB',
```

Comment:
Why there are so many color shades, are those colors really used in the project?